scene/animation/animation_tree_player.cpp

```diff
@@ -815,9 +814,9 @@ void AnimationTreePlayer::_process_animation(float p_delta) {
 					case Animation::TYPE_VALUE: { ///< Set a value in a property, can be interpolated.
 
 						if (a->value_track_is_continuous(tr.local_track)) {
-							Variant blended, value = a->value_track_interpolate(tr.local_track,anim_list->time);
-							Variant::blend(tr.track->object->get(tr.track->property),value,blend,blended);
-							tr.track->object->set(tr.track->property,blended);
+							Variant value = a->value_track_interpolate(tr.local_track,anim_list->time);
```

Comment:
The one in the class definition gets initialized back on line 769 and 770. The current `value` from the animation track gets blended onto that and then set back onto the object. Ideally the `set` calls should be moved down to step 3 so they only get set once, but I got bugs when I tried to do that before. I *think* it should work now, but I haven't analyzed all the cases yet.